### ISSUE TITLE
Property substitution modifies source files, not destination files

### ISSUE THREAD

**Issue by Freaky-namuH** [alexec/docker-java-orchestration#17]
I have some spring based projects that use property placeholder configurer to provide default properties for applications.

Property substitution is good in some cases (not in this, but I'll raise another issue), but the problem is that it doesn't substitute on the copied files, it does it on the source files.

I've trace the bug to [FileOrchestrator.java#L73](https://github.com/alexec/docker-java-orchestration/blob/20fe7e3cbeb6739582b7e003520065191b01c113/docker-java-orchestration-core/src/main/java/com/alexecollins/docker/orchestration/FileOrchestrator.java#L73)

This is best demonstrated using this test project: https://github.com/Freaky-namuH/docker-maven-test

```
git clone https://github.com/Freaky-namuH/docker-maven-test.git
cd docker-maven-test
cat arandom.properties
mvn package docker:package
cat arandom.properties
git status
cat target/docker/app/arandom.properties
```

Expected output: ```cat arandom.properties```
```
tmpdir=${java.io.tmpdir}
```

Actual output: ```cat arandom.properties```
```
tmpdir=/var/folders/n_/jzgmn1792q3gytr38hfrkn5h0000gn/T/
```

**Comment by alexec** [comment]
I've been mulling this over, as there are a couple of options:

1. Do nothing. This would make this tool unusable in many situations, so this is a reject.
2. Remove filtering on files that are "added" (i.e. delete line 73). This is a simple change, and is API breaking.
3. Expose filtering configuration. This is more complex, and provides more control, assumes users want this feature.

I'd prefer to do 2. But I can see that you might want to filter some "added" resources and not others. I think I'll do that.

**Comment by alexec** [comment]
```yml
packaging:
    add:
        - example-${project.version}.jar
        - path: foo.properties
          filter: false
```

**Comment by alexec** [comment]
```
alex-collinss-macbook:docker-maven-test alexc$ cat target/docker/app/arandom.properties 
tmpdir=${java.io.tmpdir}
```

**Comment by alexec** [comment]
Fixed in 2.4.0-SNAPSHOT. Passed your test. That was easy! Closing.
